daft/functions/misc.py

```diff
@@ -99,3 +99,12 @@ def format(f_string: str, *args: Expression | str) -> Expression:
         result = result + expr
 
     return result
+
+
+def file(expr: Expression) -> Expression:
```

Comment:
Do we want to name this `open` instead? That would be more pythonic.

Reply:
i think `file` is pretty pythonic as we're not really opening it yet. In python you have `str(), int(), list(), ...` for type conversion, so `file(str | bytes)` feels very natural to me. We can always rename or alias it later on if we get feedback that `file` is unintuitive.

Reply:
I think `file` makes sense for a function that takes some data and wraps it around a file type, which is what the other functions do. But I don't think it would be very intuitive to combine that with a function that takes in a file path. In Python, the built-in [open](https://docs.python.org/3/library/functions.html#open) function does just that.

Reply:
I understand your perspective on the naming, but honestly I think both options work fine. Mind if we ship this version and tweak it later if we need to? I'd rather not get stuck on naming when the functionality is solid.

Reply:
sounds good